weasyl/read_staff_yaml.py

```diff
@@ -1,4 +1,6 @@
 """
+from __future__ import absolute_import
```

Comment:
This will never get executed. Docstring begins at L1, and ends at L36 of this commit.